pkg/services/ngalert/schedule/registry.go

```diff
@@ -326,7 +308,6 @@ func (r ruleWithFolder) Fingerprint() fingerprint {
 	writeString(rule.Title)
 	writeString(rule.NamespaceUID)
 	writeString(r.folderTitle)
-	writeLabels(rule.Labels)
```

Comment:
Looking at Yuri's comments, we cannot remove labels as it is fundamental to the fingerprint.

Reply:
You are correct thanks.